R/write.R

```diff
@@ -70,6 +70,7 @@ write_union <- function(path, lines, quiet = FALSE) {
 write_over <- function(path, lines, quiet = FALSE) {
   stopifnot(is.character(lines), length(lines) > 0)
   path <- user_path_prep(path)
+  fs::dir_create(dirname(path))
```

Comment:
We need this because the `.github/workflows` directory likely won't exist, I don't believe it should cause any problems with existing code, we may also want to use this in `write_union()` above.

Reply:
This comes up in many places and we've basically decided to keep `write_over()` and `write_union()` as dumb as possible. I think we're pretty consistent about making the caller responsible for creating the parent directory. We use the internal helper `create_directory()` everywhere for this.

Reply:
Thanks, I made this change in 8568602b